fortran/testpar/benchmark_mdwr.F90

```diff
@@ -0,0 +1,558 @@
+! * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+!   Copyright by The HDF Group.                                               *
+!   Copyright by the Board of Trustees of the University of Illinois.         *
```

Comment:
Is this benchmark new or was it written back when the multi dataset feature was initially worked on? If it's new we should avoid the University of Illinois part of the header.